railgun/hack/generators/controllers/networking/main.go

```diff
@@ -20,103 +20,125 @@ const outputFile = "controllers/configuration/zz_generated_controllers.go"
 // when you run `make controllers`.
 var inputControllersNeeded = &typesNeeded{
 	typeNeeded{
-		PackageImportAlias: "corev1",
-		PackageAlias:       "CoreV1",
-		Package:            "k8s.io/api/core/v1",
-		Type:               "Service",
-		Plural:             "services",
-		URL:                "v1",
-		CacheType:          "Service",
+		PackageImportAlias:                "corev1",
+		PackageAlias:                      "CoreV1",
+		Package:                           "k8s.io/api/core/v1",
+		Type:                              "Service",
+		Plural:                            "services",
+		URL:                               "v1",
+		CacheType:                         "Service",
+		AcceptsIngressClassNameAnnotation: false,
+		AcceptsIngressClassNameSpec:       false,
 	},
 	typeNeeded{
-		PackageImportAlias: "corev1",
-		PackageAlias:       "CoreV1",
-		Package:            "k8s.io/api/core/v1",
-		Type:               "Endpoints",
-		Plural:             "endpoints",
-		URL:                "v1",
-		CacheType:          "Endpoint",
+		PackageImportAlias:                "corev1",
+		PackageAlias:                      "CoreV1",
+		Package:                           "k8s.io/api/core/v1",
```

Comment:
nit: `const` out the `Package`s

Reply:
:+1: e229d5a4440941faf39f7eee6d95ea47aab2a8e1 resolves